src/bigquery.ts

```diff
@@ -2113,7 +2116,10 @@ export class BigQuery extends Service {
    * ```
    */
   query(query: string, options?: QueryOptions): Promise<QueryRowsResponse>;
-  query(query: Query, options?: QueryOptions): Promise<SimpleQueryRowsResponse>;
+  query(
+    query: Query,
+    options?: QueryOptions
+  ): Promise<SimpleQueryRowsResponse> | Promise<QueryRowsResponse>;
```

Comment:
Do you need to also update the docstring to have an example of this? Or is it going to be the same?